mindjunkies/lecture/views.py

```
@@ -18,7 +18,7 @@
 from django.views.generic import TemplateView
 from django.views.generic.edit import CreateView, FormView, UpdateView
 
-from mindjunkies.courses.models import Course, Module
+from mindjunkies.courses.models import Course, Module, CourseToken
```

Ruff (0.8.2)

`mindjunkies.courses.models.CourseToken` imported but unused

Remove unused import: `mindjunkies.courses.models.CourseToken`

(F401)